client/src/api/GeoLocationService.ts

```diff
@@ -0,0 +1,23 @@
+import { IGeolocationResponse } from './types'
+
+export default class GeoLocationService {
+    static async getCurrentLocationCoords(): Promise<IGeolocationResponse> {
```

Comment:
you don't need to do it `async` if it doesn't use `await`

Reply:
corrected